### ISSUE TITLE
Getting in touch

### ISSUE THREAD

**Issue by hkjels** [mint-lang/ui#1]
Hi,
This is not an issue per se. I just would like to know more about `mint` as this looks very interesting and somewhat in line with some thoughts I've been having about creating gui's. The thing that peeks my interest, is the fact that your using `svg` for common elements. That leads me to think that animation will be a first-class citizen of the library, which is usually just an afterthought.
Am I right?
If so, is there some way I can contribute?
Btw: the language looks nice to 👍

**Comment by gdotdesign** [comment]
Hi,

Thanks for getting in touch. I take it as good sign that putting out some code out of context already got the interest of some people.

TL;DR; version of the language:
- Compiles to JavaScript
- Specifically designed for SPAs (Single Page Applications) (other platforms are possible but unlikely) 
- Strongly typed
- Expression oriented
- Somewhat functional
- **CSS** and **HTML** (and SVG) are first-class citizens
- **application state**, **routing** and **componetns** are language features instead of libraries (I'm considering adding animation here as well but I need some reasearch before I commit to specific way)
- there are language constructs to handle **synchronous  and asynchronous computations that might fail** 

The current stage includes a working **parser, type-checker, formatter, compiler and a rudamentary package manager**. I'm at a point where I am getting ready for the aplha release (that is why the repositories popped up).

I've just set up a Slack workspace for Mint, you can register with this link:
https://join.slack.com/t/mintlang/shared_invite/enQtMzI0MDQzNjMyNjkyLTE3Y2E2NTY5MTQ0YzNjYzAzNjRjMWNmN2JlOWNhZmUyYmNkYzExOTZjNWRiNmIwM2ZmODZlYTZmOGUyZGU0ZTA

I suggest we discuss the specifics on how you can contribute there :smile:
